judge/models/problem.py

```diff
@@ -100,6 +100,9 @@ class Problem(models.Model):
     name = models.CharField(max_length=100, verbose_name=_('problem name'), db_index=True,
                             help_text=_('The full name of the problem, '
                                         'as shown in the problem list.'))
+    pdf_url = models.CharField(max_length=100, verbose_name=_('PDF statement URL'), blank=True,
+                               help_text=_('URL to PDF statement. The PDF file must be embeddable (Mobile web browsers'
+                                           'may not support embedding). Fallback included.'))
     description = models.TextField(verbose_name=_('problem body'))
```

Comment:
the `description` should have `blank=true` since there are many problems that pdf is the only thing we have.